### DISCUSSION TITLE
Where does the extension take the dbt profile from?

### DISCUSSION THREAD

**Discussion by alejobs** [AltimateAI/vscode-dbt-power-user#153] [Q&A] [answered]
Hi,

I have installed the extension together with [Visual Studio Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers). Once configured, if I go to the terminal and execute:

`dbt debug`

It runs successfully.

However when I hit on any DBT extension button like "Run dbt model" or "Compiled dbt preview" it raises a connection error:

`dbt.exceptions.FailedToConnectException: Database Error
  250001 (08001): Failed to connect to DB. Verify the account name is correct: aaa.snowflakecomputing.com:443. 000403: HTTP 403: Forbidden`

**Comment by mdesmet** [comment]
dbt debug won't connect to your database. so it doesn't prove your connection details are right.

What happens if you run `dbt run --select your_model`

**Comment by alejobs** [comment]
Hi @mdesmet ,

I've tried to run a model and it is executed successfully. So, what is the difference between the terminal and what the extension runs?

Thanks.

**Reply by mdesmet** [reply]
is the account name correct in the above error? if not where does it come from?

**Reply by alejobs** [reply]
I don't have any idea.... That's why I was asking

**Reply by alejobs** [reply]
So I mean, the account name is NOT correct. But if I run `dbt run` in the terminal it gets correctly the account name

**Comment by alejobs** [comment]
As I said, whenever I execute through the terminal everything runs fine. Even I've tried to run dbt-osmosis through the terminal and executed it successfully. Can anyone let me know where does the extension take the credentials from?

Thanks

**Comment by mdesmet** [comment, marked as answer]
The fact that your terminal is working doesn't mean that vscode is correctly configured. It might be that you have selected another Python environments or that environment variables are not known by vscode itself or even configured differently

the `aaa` is definitely not coming from the extension but should be something configured in your environment.

**Comment by alejobs** [comment]
At the end it seemed my issue was the same as: #97
